termennetwerk.ts

```diff
@@ -1,24 +1,38 @@
 import { GraphQLClient, gql } from "graphql-request";
+import { ColumnRefinement } from "@pldn/ldwizard/types/WizardConfig";
 
 const endpoint = "https://termennetwerk-api.netwerkdigitaalerfgoed.nl/graphql";
 
-interface GraphQlResponse {
+interface Term {
+  uri: string;
+  prefLabel: string[];
+  altLabel: string[];
+}
+
+interface Result {
+  __typename: string;
+  terms: Term[];
```

Comment:
This does not seem to account for error conditions. If the terminology source returns an error, the GraphQL response [looks different](https://github.com/netwerk-digitaal-erfgoed/network-of-terms/tree/master/packages/network-of-terms-graphql#query-a-single-source).

Reply:
Good point - I will add the error conditions, do you have a pointer to code I can reuse?

Reply:
Perhaps the demonstrator code is a good starting point: https://github.com/netwerk-digitaal-erfgoed/network-of-terms-demo/blob/4d03e014698c4605b9c03f60ccea072fd0d6abdd/src/query.ts#L5

Reply:
Thanks, that is helpful! Should we consider making these types part of the network-of-terms package so there is no need to define these every time, or is this too client specific?

Reply:
On second thought: the code checks for error conditions through the `__typename` contents before the Term results are addressed, so I prefer to keep current interface definitions for the sake of complexity.